.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Python Tests
+
+# 1. Run on every push 
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    # 2. Use the latest Ubuntu runner
+    runs-on: ubuntu-latest
+
+    # 3. Spin up a PostgreSQL service for tests
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test_db
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd "pg_isready -U postgres"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      # 4. Check out code
+      - uses: actions/checkout@v3
+
+      # 5. Wait for Postgres to be ready
+      - name: Wait for Postgres
+        run: |
+          until pg_isready -h localhost -p 5432 -U postgres; do
+            echo "Waiting for Postgres…"
+            sleep 1
+          done
```

Comment:
Another wait step here shouldn't be necessary. [If I understand the docs correctly](https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/creating-postgresql-service-containers#running-jobs-in-containers), Github Actions will wait until `health-cmd` is successful before continuing to `steps`